### PR TITLE
add power readings to ac_config

### PR DESCRIPTION
**Pull request by dacrypt**
This is to address #75

### CODE REVIEW

**Comment by sampsyo** [comment]
Looks pretty cool overall! Any chance you could try giving these two properties more informative names? TBH I have no idea what “inout total power” means... or what units they’re in. We can pick more meaningful names than LG did!

**Comment by gladhorn** [comment]
The flake8 issue should be fixed now :)

**Comment by dacrypt** [comment]
@sampsyo is that ok or should I return just the value and not the whole json returned by lg?

**Comment by sampsyo** [comment]
Yeah, absolutely; returning an actual integer instead of the JSON object would definitely be more useful. In general, the idea with wideq.client is to be a sensible wrapper that programmers would actually want to use—not to faithfully reflect the LG API, which is usually pretty incomprehensible.

Is that what the current state of the code does, or do we need to add a field access to those methods?

**Comment by dacrypt** [comment]
ok, that should do it. let me know

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 .tox
 dist/
 wideq_state.json
+wideq/__pycache__
```
Comment (sampsyo) [inline .gitignore]: Maybe just `__pycache__`? We can ignore it wherever it appears.

Comment (dacrypt) [reply .gitignore]: updated

**Comment by dacrypt** [comment]
> Looks good! One tiny suggestion within. I can't test this because my units don't support it, but if you have tested and it looks good to you, let's merge.

it seems to be reporting fine. You can go ahead and merge it

**Comment by sampsyo** [comment]
Awesome.

**Pull request merged**